### PR TITLE
[SMALL] RevEng: Fixing missing character in namespace…

### LINKED ISSUE #3228: RevEng: Namespace is missing first character
The following command...
```ps1
Scaffold-DbContext -OutputDirectory Models 'Data Source=(localdb)\MSSQLLocalDB;Initial Catalog=Northwind;Integrated Security=True' EntityFramework.SqlServer
```
...outputs code with the following namespace.
```C#
namespace ConsoleApplication1.odels
{
    // ...
}
```

### PR DESCRIPTION
**Pull request by lajones**
...when defining  output directory through Powershell.

Fixes #3228. Problem was that Powershell sends the project path with trailing "\" and we were expecting without.

### CODE REVIEW

File: src/EntityFramework.Relational.Design/ReverseEngineering/ReverseEngineeringGenerator.cs [hunk 1]
```diff
@@ -127,17 +121,34 @@ public virtual IModel GetMetadataModel([NotNull] ReverseEngineeringConfiguration
 
         private static char[] directorySeparatorChars = new char[] { Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar };
 
-        private string ConstructNamespace(string rootNamespace, string projectPath, string relativePath)
+        private string ConstructNamespace(string rootNamespace, string projectPath, string outputPath)
```
Comment (lajones) [inline src/EntityFramework.Relational.Design/ReverseEngineering/ReverseEngineeringGenerator.cs]: Anyone mind if I make this public and add some unit tests?

Comment (natemcmaster) [inline src/EntityFramework.Relational.Design/ReverseEngineering/ReverseEngineeringGenerator.cs]: Nope, go ahead. This is going to be moved in to commands anyways when we do the big RelationalDesign => Scaffolding rename.

**Comment by natemcmaster** [comment]
:shipit:

**Comment by lajones** [comment]
Checked in with commit b9be76d.

**Pull request closed without merging**